administration/hot-reload.md

```diff
@@ -45,6 +45,22 @@ Hot reloading also can be kicked via `SIGHUP`.
 
 `SIGHUP` signal is not supported on Windows. So, users can't enable this feature on Windows.
 
+## How to confirm reloaded or not
+
+### via HTTP
+
+The number of hot reloaded count can be obtained via the HTTP endpoint that is:
+
+* `GET /api/v2/reload`
+
+The endpoint returns the count of hot-reloaded as follows:
+
+```json
+{"hot_reload_count":3}
```

Comment:
Do we get this in the "normal" metrics too? I think it is useful from a monitoring perspective.

Reply:
No, we don't. For monitoring perspective, we can retrieve via `GET /api/v2/metrics/[prometheus]` endpoints.

Reply:
You've confused me now - do we have this metric in the prometheus metrics?

Reply:
We'll have this metrics in Prometheus metric after merged fluent-bit#7899

Reply:
Ah, I should say yes for your question. Sorry for confusion🥲